frontend/src/routes/Governance/discovered/DiscoveredPolicies.tsx

```diff
@@ -0,0 +1,214 @@
+import { DOC_LINKS } from '../../../lib/doc-util'
+import { TechPreviewAlert } from '../../../components/TechPreviewAlert'
+import { discoveredSourceCell, policyViolationSummary, severityCell } from './ByCluster/common'
+import { ClusterPolicyViolationIcons2 } from '../components/ClusterPolicyViolations'
+import { AcmTableWithEngine } from '../common/AcmTableWithEngine'
+import { uniq } from 'lodash'
+
+export default function DiscoveredPolicies() {
+  const { isFetching, data, err } = useFetchPolicies()
+  const { t } = useTranslation()
+
+  const discoveredPoliciesCols = useMemo<IAcmTableColumn<DiscoverdPolicyTableItem>[]>(
+    () => [
+      {
+        header: t('Name'),
+        cell: (item: DiscoverdPolicyTableItem) => (
+          <Link
+            to={generatePath(NavigationPath.discoveredByCluster, {
+              kind: item.kind,
+              policyName: item.name,
+              apiGroup: item.policies[0].apigroup,
+              apiVersion: item.policies[0].apiversion,
+              policyNamespace: item.policies[0].cluster,
+            })}
+            state={{
+              from: NavigationPath.policies,
+            }}
+          >
+            {item.name}
+          </Link>
+        ),
+        // Policy name
+        sort: 'name',
+        search: 'name',
+        id: 'name',
+        exportContent: (item) => item.name,
+      },
+      {
+        header: t('Engine'),
+        cell: (item: DiscoverdPolicyTableItem) => getEngineWithSvg(item.kind),
+        sort: (a: DiscoverdPolicyTableItem, b: DiscoverdPolicyTableItem) =>
+          compareStrings(getEngineString(a.kind), getEngineString(b.kind)),
+        search: (item: DiscoverdPolicyTableItem) => getEngineString(item.kind),
+        id: 'engine',
+        exportContent: (item: DiscoverdPolicyTableItem) => getEngineString(item.kind),
+      },
+      {
+        header: t('Kind'),
+        cell: 'kind',
+        sort: 'kind',
+        search: 'kind',
+        id: 'kind',
+        exportContent: (item: DiscoverdPolicyTableItem) => item.kind,
+      },
+      {
+        header: t('Response action'),
+        cell: 'responseAction',
+        sort: 'responseAction',
+        search: 'responseAction',
+        id: 'responseAction',
+        tooltip: t(
+          'For Open Cluster Management policies, response action refers to the remediationAction. For Gatekeeper constraints, response action refers to the enforcementAction.": "For Open Cluster Management policies, response action refers to the remediationAction. For Gatekeeper constraints, response action refers to the enforcementAction.'
+        ),
+        exportContent: (item: DiscoverdPolicyTableItem) => item.responseAction,
+      },
+      {
+        header: t('Severity'),
+        // TODO Add severity icon
+        cell: severityCell,
+        sort: 'severity',
+        search: 'severity',
+        id: 'severity',
+        exportContent: (item) => item.severity,
+      },
+      {
```

Comment:
It seems that this default column order isn't the same here as in `ByClusterCols`. Could you please make them consistent?